packages/devextreme/testing/tests/DevExpress.ui.widgets/multiView.tests.js

```diff
@@ -1659,3 +1659,200 @@ QUnit.module('swipeable disabled state', () => {
+            items: [
+                { text: '1', visible: true },
+                { text: '2', visible: false },
+                { text: '3', visible: true },
+            ],
+            selectedIndex: 0
+        });
+        const instance = $multiView.dxMultiView('instance');
+        instance.option('items[1].visible', true);
+
+        assert.strictEqual(instance.option('selectedIndex'), 0, 'selectedIndex is updated on proper index');
+    });
+
+    QUnit.test('when hiding last visible item selectedIndex should return to 0 index', function(assert) {
+        const $multiView = $('#multiView').dxMultiView({
+            items: [
+                { text: '1', visible: false },
+                { text: '2', visible: false },
+                { text: '3', visible: true },
+            ],
+            selectedIndex: 2
+        });
+        const instance = $multiView.dxMultiView('instance');
+        instance.option('items[2].visible', false);
+
+        assert.strictEqual(instance.option('selectedIndex'), 0, 'selectedIndex is updated on proper index');
+    });
+
+    QUnit.test('next visible item should be selected if currently selected item is hidden and loop=true', function(assert) {
+        const $multiView = $('#multiView').dxMultiView({
+            items: [
+                { text: '1', visible: false },
+                { text: '2', visible: false },
+                { text: '3', visible: true },
+            ],
+            loop: true
+        });
+        const instance = $multiView.dxMultiView('instance');
+
+        assert.strictEqual(instance.option('selectedIndex'), 2, 'selectedIndex is updated on proper index');
+    });
+
+    QUnit.test('next visible item should be selected if currently selected item is hidden and loop=true | items=[{ visible: true }, { visible: false }], selectedIndex = 1', function(assert) {
+        const $multiView = $('#multiView').dxMultiView({
+            items: [
+                { text: '1', visible: true },
+                { text: '2', visible: false }
+            ],
+            selectedIndex: 1,
+            loop: true
+        });
+        const instance = $multiView.dxMultiView('instance');
+
+        assert.strictEqual(instance.option('selectedIndex'), 0, 'selectedIndex is updated on proper index');
+    });
+
+    QUnit.test('previous visible item should be selected if currently selected item is hidden and multiple hidden item is in between them | loop = true', function(assert) {
+        const $multiView = $('#multiView').dxMultiView({
+            items: [
+                { text: '1', visible: true },
+                { text: '2', visible: false },
+                { text: '3', visible: false }
+            ],
+            selectedIndex: 2,
+            loop: true
+        });
+        const instance = $multiView.dxMultiView('instance');
+
+        assert.strictEqual(instance.option('selectedIndex'), 0, 'selectedIndex is updated on proper index');
+    });
```

Comment:
Yeah, so that's the configuration which we should use to check that our feature works well with looping)
But let's remove other similar tests and make this test's naming simpler
"next visible item should be selected if currently selected item is hidden and loop=true" is good